components/home/header-mobile/index.js

```diff
@@ -13,27 +15,83 @@ const HomeHeaderMobile = () => {
   };
 
   return (
-    <div className="c-header-mobile">
+    <div className={cx('c-header-mobile', { '-open': isOpen } )}>
       <p className="header-description">2020 by <a href="https://www.vizzuality.com/" target="_blank" rel="noopener noreferrer">
         vizzuality.</a>
       </p>
 
       <button className="header-menu--button" onClick={handleMenu}>
-        <Icon name={isOpen ? "menu-close" : "menu"} className="-medium -light" />
+        <Icon name={isOpen ? "menu-close" : "menu"} className="-medium -dark" />
 
       </button>
       {isOpen && (
         <div className="menu-content">
-          <Download className="-mobile menu-item" />
-          <SocialMedia className="-mobile menu-item" />
-          <a
-            href="https://www.vizzuality.com/privacy-policy"
-            title="Privacy policy"
-            target="_blank"
-            className="privacy-link menu-item"
-          >
-            Privacy policy.
-          </a>
+          <ul className="menu">
+            <li className="menu-item">
+              <h2 className="-block0">Building knowledge.</h2>
+              <div>
+                <Link href={{ pathname: '/report/0/[slug]', query: { slug: '0' } }}>
```

Comment:
This won't work as the rendered page on the link its not exactly the same as the one rendered by the modal. You can use a router.push as we have on the pieces or we can save the progress in sessionStorage or similar